README.md

```diff
@@ -304,3 +304,17 @@ export function TestSafeAreaProvider() {
   );
 }
 ```
+
+### Testing With Enzyme
+
+If you are using [`enzyme`](https://github.com/enzymejs/enzyme) and testing a component that uses
+`useSafeAreaInsets` or other hooks requiring the `SafeAreaProvider`, you may need to provide
+`initialSafeAreaInsets` to ensure the `SafeAreaProvider` renders its children:
+
+```js
+const wrapper = shallow(
+  <SafeAreaProvider initialSafeAreaInsets={{ top: 0, left: 0, right: 0, bottom: 0 }}
+    <ComponentUnderTest />
+  </SafeAreaProvider>
+)
+```
```

Comment:
Not sure if it's worth linking to the appropriate issues/comments in here?

Reply:
`initialSafeAreaInsets` is the old API. Aren't you trying to do the same thing as the previous example under testing? What the previous example seems to miss is to render a component inside the provider which you have added here. Can you update the other example to include that instead?

Reply:
> Can you update the other example to include that

Absolutely! I was wondering what was going on in that example because it didn't make sense to render `SafeAreaProvider` with nothing under it but it seems like it was maybe just an incomplete example?

> `initialSafeAreaInsets` is the old API.

Ah, didn't know that. Should I include it in the "Deprecated apis" section so it's more visible?

Reply:
Yeah I think it's just incomplete example. cc @janicduplessis

Reply:
In the meantime, I've updated the example [here](f79be7369b0a2e0e33981a0eb82d01e803251720) and added `initialSafeAreaInsets` to the deprecated apis list [here](f6d4ee1578c222d30438230e16cd6395d4a84130) 👍

Reply:
The deprecated api is actually initialWindowSafeAreaInsets, and is already included in the doc so I just removed it. Rest looks good, thanks!